### ISSUE TITLE
[BUG] pyradio-0.9.3 cannot be executed.

### ISSUE THREAD

**Issue by amano-kenji** [coderholic/pyradio#231]
**Describe the bug**

```
self.xdg.xdg_compliant = False
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.11/pyradio", line 8, in <module>
    sys.exit(shell())
             ^^^^^^^
  File "/usr/lib/python3.11/site-packages/pyradio/main.py", line 383, in shell
    read_config(pyradio_config)
  File "/usr/lib/python3.11/site-packages/pyradio/main.py", line 918, in read_config
    pyradio_config.migrate_xdg()
  File "/usr/lib/python3.11/site-packages/pyradio/config.py", line 2016, in migrate_xdg
    self._copy_icon()
  File "/usr/lib/python3.11/site-packages/pyradio/config.py", line 264, in _copy_icon
    with open(upd, 'w') as f:
         ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.local/state/pyradio/UPDATE_ICON'
```

**System (please complete the following information):**
 - OS: Gentoo Linux
 - **Python** version: 3.11
 - **PyRadio** Version: 0.9.3

**Additional context**

In pyradio/config, I have xdg_compliant = True.

Migration should be optional. Not everyone migrates from older versions. I removed old files except radio stations before executing pyradio.

**Comment by amano-kenji** [comment]
This turned out to be an issue with my firejail profile. I fixed the issue by updating pyradio firejail profile.

**Comment by s-n-g** [comment]
No, this has nothing to do with your firejail profile; it happens on a new installation, when `xdg_compliant = True` and `~/.local/state/pyradio` does not yet exist...

A workaround would be to:

```
mkdir ~/.local/state/pyradio
pyradio
```

Reopening until 0.9.3.1 is out...

**Comment by s-n-g** [comment]
fixed in **0.9.3.1**
